torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -168,9 +168,12 @@ ParallelTypeBitmap getParallelBroadcastDomains(
 
 namespace loop_utils {
 
+// TODO: Clean this up, Naoya added a mechanism we should be able to reuse.
 std::pair<kir::ForLoop*, int64_t> getAllocPoint(
```

Comment:
@naoyam could you please double check the logic in here, worried I may be messing up the alloc point like I did with the lower alloc pass.

Reply:
This looks good to me.